python/planout/ops/core.py

```diff
@@ -90,6 +91,15 @@ def pretty(self):
         return f
 
 
+class Map(PlanOutOpSimple):
+
+    def simpleExecute(self):
+        m = deepcopy(self.args)
+        del m['op']
+        del m['salt']
```

Comment:
args['salt'] would only be present for random ops. Right?

Or perhaps it is coming from here(?)
```python
        # if the value is operator, add the name of the variable as a salt if no
        # salt is provided.
        if ops.Operators.isOperator(value) and 'salt' not in value:
            value['salt'] = var
```